doc/interfaces/supercomputer_data/regions.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Regions"
+filters:
+  - "include-code-files"
+---
+
+
+
+## Regions
+
+Any region that can be selected for viewing on the map or plot.
+Includes Super Regions and Sub Regions.
+
+- All regions (Super Regions and Sub Regions) **MUST** have a unique numeric region
+  identifier.
+    - Regions and Region Collections **MUST NOT** be identified by any of the following
+      reserved strings:
+        - `undefined`
+        - `regions`
+        - `collections`
+        - `metadata`
+
+
+### Super Regions
+
+These are top-level regions, e.g. "Western U.S.", "Canada".
+The webapp needs to be able to quickly load the list of super-regions so it can display
+a selector to the user. Depending on which is selected, different sub-regions may
+display.
+
+- Super Regions definition file **MUST** be named `root.json`.
+- Super Regions definition file **MUST** be in compliance with the schema.
+- Super Regions definition file **MUST** be pushed to the `regions/` subdirectory of the
+  incoming directory, e.g. `{incomingDir}/regions/root.json`.
+- Super Regions **MAY** be omitted from the Super Regions definition file if no variable
+  are visible for the super region.
+
+  :::{.callout-note}
+  In a future version of the schema, we should offer a boolean config option `enabled`
+  and recommend setting it to false to omit a region.
+  :::
+
+- Super Regions with no Sub Regions/Collections **MAY** be included in the Super Regions
+  definition file.
+
```

Comment:
- Each super regions has the same variables than its related sub regions, and plots are presented the same way.

Reply:
d99377e